src/crate/theme/vendor/rtd_compat/extension.py

```diff
@@ -1,255 +0,0 @@
-        # 'proxied_api_host': '{{ project.proxied_api_host }}',
-
-        'github_user': github_user,
-        'github_repo': github_repo,
-        'github_version': os.environ.get("READTHEDOCS_GIT_IDENTIFIER"),
-        'display_github': github_user is not None,
-        'bitbucket_user': bitbucket_user,
-        'bitbucket_repo': bitbucket_repo,
-        'bitbucket_version': os.environ.get("READTHEDOCS_GIT_IDENTIFIER"),
-        'display_bitbucket': bitbucket_user is not None,
-        'gitlab_user': gitlab_user,
-        'gitlab_repo': gitlab_repo,
-        'gitlab_version': os.environ.get("READTHEDOCS_GIT_IDENTIFIER"),
-        'display_gitlab': gitlab_user is not None,
-        'READTHEDOCS': True,
-        'using_theme': (config.html_theme == "default"),
-        'new_theme': (config.html_theme == "sphinx_rtd_theme"),
-        'source_suffix': ".rst",
-        'ad_free': False,
-        'docsearch_disabled': False,
-
-        # We don't support Google analytics anymore.
-        # See https://github.com/readthedocs/readthedocs.org/issues/9530
-        'user_analytics_code': "",
-        'global_analytics_code': None,
-
-        'commit': os.environ.get("READTHEDOCS_GIT_COMMIT_HASH")[:8],
-    }
-
-    # For sphinx >=1.8 we can use html_baseurl to set the canonical URL.
-    # https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_baseurl
-    if version_info >= (1, 8):
-        if not hasattr(config, 'html_baseurl'):
-            config.html_baseurl = context['canonical_url']
-        context['canonical_url'] = None
-
-
-    if hasattr(config, 'html_context'):
-        for key in context:
-            if key not in config.html_context:
-                config.html_context[key] = context[key]
-    else:
-        config.html_context = context
-
-    project_language = os.environ.get("READTHEDOCS_LANGUAGE")
-
-    # User's Sphinx configurations
-    language_user = config.language
-    latex_engine_user = config.latex_engine
-    latex_elements_user = config.latex_elements
-
-    # Remove this once xindy gets installed in Docker image and XINDYOPS
-    # env variable is supported
-    # https://github.com/rtfd/readthedocs-docker-images/pull/98
-    latex_use_xindy = False
-
-    chinese = any([
-        language_user in ('zh_CN', 'zh_TW'),
-        project_language in ('zh_CN', 'zh_TW'),
-    ])
-
-    japanese = any([
-        language_user == 'ja',
-        project_language == 'ja',
-    ])
-
-    if chinese:
-        config.latex_engine = latex_engine_user or 'xelatex'
-
-        latex_elements_rtd = {
-            'preamble': '\\usepackage[UTF8]{ctex}\n',
-        }
-        config.latex_elements = latex_elements_user or latex_elements_rtd
-    elif japanese:
-        config.latex_engine = latex_engine_user or 'platex'
-
-    # Make sure our build directory is always excluded
-    if not hasattr(config, "exclude_patterns"):
-        config.exclude_patterns = []
-    config.exclude_patterns.extend(['_build'])
```

Comment:
That also looks interesting. We can add it directly to our theme's configuration.